AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -48,6 +48,9 @@ jiji.com,edp24.co.uk,technologyreview.com,inews.co.uk,nordbayern.de,thelocal.ch,
 !
 ! SECTION: Subscriptions - Regular rules
 !
+icij.org##.home-banner-newsletter
+icij.org##.newsletter-sidebar
```

Comment:
cannot reproduce this element

Reply:
`.home-banner-newsletter`
https://www.icij.org/
<img width="1440" alt="Screenshot 2023-03-30 at 12 55 59 PM" src="https://user-images.githubusercontent.com/110956608/228909805-a8a025da-d04d-42ea-b67a-cc1d6c78ec95.png">

`.newsletter-sidebar`
https://www.icij.org/investigations/deforestation-inc/canadian-lawmakers-call-for-probe-into-pulp-and-paper-giant-following-deforestation-inc-revelations/
<img width="1440" alt="Screenshot 2023-03-30 at 12 57 14 PM" src="https://user-images.githubusercontent.com/110956608/228910060-bf52c4b0-2f5e-4f43-93ef-da40ab293f29.png">